Dockerfile

```diff
@@ -27,8 +27,12 @@ ENV P4C_DEPS bison \
              python-ipaddr \
              python-pip \
              python-setuptools \
-             tcpdump \
-             libpcap-dev
+             tcpdump
+ENV P4C_EBPF_DEPS libpcap-dev \
```

Comment:
Please update the ebpf/README.md file to list all these dependences.
You may need a specific version of llvm.